src/Common/tests/System/IO/Compression/ZipTestHelper.cs

```diff
@@ -198,7 +183,7 @@ public static void IsZipSameAsDir(Stream archiveFile, String directory, ZipArchi
                         if (entry == null) //entry not found
                         {
                             string entryNameOtherSlash = FlipSlashes(entryName);
-                            Boolean isEmtpy = !allFilesInDir.Any(
+                            Boolean isEmtpy = !files.Any(
```

Comment:
Nit: `bool`. Or maybe the code formatting cleanup should be done separately since it looks like a lot of the test and product code is using type names instead of language keywords throughout.